recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
```

Comment:
Looking at this [PR](https://github.com/conda-forge/earthengine-api-feedstock/pull/170/files), there's a good chance that we don't need to bump the build number. Bumping the build number produces a new version of the library which is publicly visible, so it would be less noisy to skip this.

Reply:
Done.